codalab/model/bundle_model.py

```diff
@@ -2481,12 +2481,12 @@ def add_user(
 
             time_quota = (
                 self.default_user_info['edu_time_quota']
-                if email.endswith(".edu")
+                if email.endswith(".edu") or email.endswith(".edu.cn")
```

Comment:
Can we pull this out into a separate function `is_academic_email` and generalize to `.edu.*`?